libs/service-portal/access-control/delegations/src/components/access/AccessItemHeader.tsx

```diff
@@ -13,7 +13,7 @@ export const AccessItemHeader = ({
   const { formatMessage } = useLocale()
 
   return (
-    <Hidden below="md">
+    <Box display={['none', 'block']} aria-hidden>
```

Comment:
```suggestion
    <Box display={['none', 'none', 'block']} aria-hidden>
```

Small regression in mobilish view.
<img width="559" alt="image" src="https://user-images.githubusercontent.com/112904566/202153912-255cf3d3-b32f-4794-b018-b8a17fdfbca3.png">